pkg/client/keycloak/config.go

```diff
@@ -103,7 +127,18 @@ func (kc *KeycloakConfig) ReadFiles() error {
 	if err != nil {
 		return err
 	}
-
+	err = shared.ReadFileValueString(kc.OSDClusterIDPRealm.ClientSecretFile, &kc.OSDClusterIDPRealm.ClientSecret)
+	if err != nil {
+		return err
+	}
+	err = shared.ReadFileValueString(kc.RedhatSSORealm.ClientIDFile, &kc.RedhatSSORealm.ClientID)
+	if err != nil {
+		return err
+	}
+	err = shared.ReadFileValueString(kc.RedhatSSORealm.ClientSecretFile, &kc.RedhatSSORealm.ClientSecret)
+	if err != nil {
+		return err
+	}
```

Comment:
I think we should be only be reading RH sso secrets if the provider is set to redhat_sso.

Reply:
Good point

Reply:
The same could be done for the mas_sso provider: https://github.com/akoserwal/kas-fleet-manager-1/blob/3a44f1500d408a7163b7f6b0967234e779e7c0f3/pkg/client/keycloak/config.go#L114-L133

WDYT? it could be something for future improvement

Reply:
yes, agreed !!